e2e/LongHaul/prerequisites/LongHaulSetup.ps1

```diff
@@ -245,6 +255,7 @@ $longhaulDeviceConnectionString = az iot hub device-identity connection-string s
 $keyvaultKvps = @{
     # Environment variables for IoT Hub E2E tests
     "IOTHUB-LONG-HAUL-DEVICE-CONNECTION-STRING" = $longhaulDeviceConnectionString;
+    "IOTHUB-LONG-HAUL-MODULE-CONNECTION-STRING" = $longhaulDeviceConnectionString;
```

Comment:
Uses the same connection string as the device.

Is this meant to be an Edge module or a device module? How it authenticates is different in either case.

Reply:
FWIW, I think we should prioritize edge modules for long first.